app/pkg/logparser/helpers_test.go

```diff
@@ -0,0 +1,113 @@
+package logparser
+
+import (
+	"testing"
+)
+
+func TestReadLogFileRaw(t *testing.T) {
+	type args struct {
+		dirPath string
+		lines   string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name: "TestReadLogFileRaw no such directory",
+			args: args{
+				dirPath: "test",
+				lines:   "10",
+			},
+			wantErr: true,
+		},
+		{
+			name: "TestReadLogFileRaw",
+			args: args{
+				dirPath: "../../tmp/device-name/cputemp",
```

Comment:
Can you please double check this path? where is the `../../tmp/` folder created?

Reply:
in the app folder, but I found that the go testing package reads the paths starting from itself (logparser/helpers_test.go), and the `../../tmp/` will always exist because I added it to the `git` repo

Reply:
In your opinion is it possible to use a `testdata` folder (see this [example](https://dave.cheney.net/2016/05/10/test-fixtures-in-go) and this [blog post](https://ieftimov.com/posts/testing-in-go-fixtures/))? I think it's a more standard approach

Reply:
I moved the needed data into `testdata` directory in the same directory with tests, but not sure if this is the right way

Reply:
Can you please describe what your doubts are? That way it will be easier for both of us to analyze the problem